go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/ONSdigital/dp-population-types-api
 
-go 1.19
+go 1.18
```

Comment:
any reason for the rollback?

Reply:
go mod was not playing nice with the new version, but also all of the rest of the infrastructure was 18 like dockerfiles so I thought just take it back